lib/wasi/src/syscalls/wasix/callback_signal.rs

```diff
@@ -0,0 +1,54 @@
+use super::*;
+use crate::syscalls::*;
+
+// FIXME: remove , since it's no longer used.
+/// ### `callback_signal()`
```

Comment:
So this need to be removed? Shall I do it?

Reply:
Yes, please go ahead

Reply:
No this is needed for the multithreading code, we can only remove it after we move over to the new threading spec which can not be done till.its stabilized

Reply:
This sets the name of the exported function that should be invoked when a signal is sent to a process, it is called by libc

Reply:
Removed the FIXME.